tests/tests/test-rpc/test-starknet-rpc.ts

```diff
@@ -531,10 +531,10 @@ describeDevMadara("Starknet RPC", (context) => {
 
       const latestBlock = await providerRPC.getBlockHashAndNumber();
 
+      // TODO: Add real values
```

Comment:
add them

Reply:
in another PR that's not related to mine just added the TODO